### DISCUSSION TITLE
能否自托管静态资源呢    JsDelivery太慢了

### DISCUSSION THREAD

**Discussion by wzydale** [AlistGo/alist#327] [Q&A] [answered]
![Snipaste_2022-01-13_02-03-43](https://user-images.githubusercontent.com/32425043/149196748-56947e01-03d5-4bb7-a0a1-2358750da6cf.jpg)
可以看到 这几个JS文件加载时间超过了10秒，能否自己指定静态资源的路径呢？ 把JS和CSS托管到国内的OSS等去使用

**Comment by xhofe** [comment, marked as answer]
可以使用本地静态资源，在配置文件中将local改为true就行。

至于放到oss因为是编译时指定的所以可能需要自行编译才可以。

**Reply by yn-ayw** [reply]
在配置文件中将local改为true就行。  配置文件里都没有这个啊

**Reply by xhofe** [reply]
看文档。
